tools/licenses/README

```diff
@@ -1,8 +1,5 @@
-To update the sky_engine LICENSE file, run the following in this directory:
+To update the sky_engine LICENSE file, run the following from the build root:
 
 ```
-pub get
-gclient sync
-dart lib/main.dart ../../.. > ../../sky/packages/sky_engine/LICENSE
-dart --checked lib/main.dart ../../.. > ../../travis/licenses.golden
+./flutter/tools/licenses/update_licenses.sh
```

Comment:
There is one more step: If the diff actually indicated that an entire license was added or removed, you need to run the license script again in release mode (it has a flag) and pipe the output into `sky/packages/sky_engine/LICENSE`